### DISCUSSION TITLE
提供OpenAI API之后，经常响应出错

### DISCUSSION THREAD

**Discussion by BlackDuckDuck** [nextai-translator/nextai-translator#648] [Q&A]
我选择的服务商是OpenAI，添加了key，其他都是默认。这种情况下，直接点插件图标的翻译，会报错。但是如果选中网页，有时候就可以翻译成功的，有时候也同样报错。这是什么原因啊？怎么解决，求大佬指点。

![image](https://user-images.githubusercontent.com/129487271/235042816-1c9419ac-c1f0-4845-a52c-5fa414e6c8c2.png)

**Comment by wuya1019** [comment]
我也遇到了同样的问题 很烦躁
